internal/backend/mqttpubsub/backend.go

```diff
@@ -101,6 +105,11 @@ func NewBackend(c BackendConfig) (*Backend, error) {
 	opts.SetOnConnectHandler(b.onConnected)
 	opts.SetConnectionLostHandler(b.onConnectionLost)
 
+	if b.config.LastWillPayload != "" {
```

Comment:
Given that the following payload would be used:

```json
{
    "mac": "0102030405060708"
}
```

Then the last-will could always be set :-)

Reply:
Can you please help me with the payload message? How can I find the mac address? thanks!